app/views/layouts/application.html.erb

```diff
@@ -50,6 +49,8 @@
 
   <div id="global-app-error" class="app-error hidden"></div>
 
+  <%= javascript_include_tag 'application' %>
```

Comment:
Can you explain the reason for moving this line? Seems fine, just curious.

Reply:
Good question... I *think* I moved it because otherwise the cookie banner Javascript didn't seem to initialise and therefore the cookie banner was just never shown. I assumed this was happening because the JS was being included before the cookie banner.